consensus/ibft/consensus_backend.go

```diff
@@ -138,8 +138,15 @@ func (i *backendIBFT) buildBlock(snap *Snapshot, parent *types.Header) (*types.B
 	// set the timestamp
 	header.Timestamp = uint64(time.Now().Unix())
 
-	// we need to include in the extra field the current set of validators
-	putIbftExtraValidators(header, snap.Set)
+	var parentCommittedSeal [][]byte
+
+	if parent.Number >= 1 {
```

Comment:
Just a suggestion, maybe to have util method isGenesisBlock(blockNumber). It's a bit more descriptive

Reply:
Added at 83e82a3e529746c8a3fc7b8437e78f6abe8136a0.

Reply:
Thanks